content/hardware/04.pro/boards/portenta-x8/tutorials/image-building/content.md

```diff
@@ -0,0 +1,208 @@
+difficulty: advanced
+tags:
+  - Embedded Linux
+  - Building
+  - Yocto-project
+author: 'Pablo Marquínez'
+hardware:
+  - hardware/04.pro/boards/portenta-x8
+---
+
+## Overview
+
+In this tutorial you will learn how to build an image for the Portenta X8 with the source code provided at our GitHub repository lmp-manifest.
+Building your image locally is helpful to quickly debug certain aspects of the systems, like bootloader or kernel support.
+
+***Please keep in mind that images built locally cannot register to a FoundriesFactory and will not be OTA compatible, but this is a good alternative for those who do not have a FoundriesFactory subscription.  This tutorial targets the customers that are not FoundriesFactory subscribers and want to extend the functionality provided by the Arduino pre-built sources by building their own images. For FoundriesFactory subscribers, we strongly suggest to make use of the Factory continuous integration system for creating your images.***
+
+## Goals
+- Build a "builder" Docker image
+- Get the required files
+- Configure the building settings
+- Build the image
+- Save the needed files for flashing
+
+### Required Hardware and Software
+
+- [Docker Engine](https://docs.docker.com/engine/install/)
+
+## Instructions
+
+### Docker
+#### Build the Docker Image
+
+You will create an image with all the needed dependencies so it will be ready to build your image.
+
+To do so you will need to clone our [lmp-manifest repository](https://github.com/arduino/lmp-manifest):
+1. Clone the lmp-manifest repository
+  ```
+  git clone https://github.com/arduino/lmp-manifest.git
+  ```
+  ![Cloning lmp-manifest repository](assets/git_clone_lmp-manifest.png)
+2. Build the Docker Image:
+  ```
+  cd lmp-manifest
+  docker build -t yocto-build ./lmp-manifest
+  ```
+  ![Building a Docker Image](assets/docker_build.png)
+
+#### Run The Docker Image (Builder)
+You will be running the image with the `-v` argument to add a volume, this will make the second drive storage unit available inside the virtual machine, so we will be able to store all the data safely.
+
+***If you do not use a volume while running the image, you will lose the data when the image stops***
+
+Run the `yocto-build` builder image with:
+```
+docker run -v <source>:/dockerVolume -it yocto-build bash
+```
+
+Switch to the `builder` user, the password is **builder**:
+```
+su builder
+```
+
+### Setup and Build
+
+***You can download a bash script that wraps all the upcoming steps [here](assets/portenta-x8_build.sh)***
+
+#### Setup the Environment
+Now that you are running inside the Docker Image you are already provided with some tools like **git-repo** which has been installed on the image building process, this was the providing process on the previous section.
+
+First of all set up the git config with your credentials, they don't need to be the real ones, they are just needed to pull the data from the git-repo.
```

Comment:
```suggestion
First configure git with your credentials. They don't need to be the real ones, but are required by `git-repo` to pull.
```
Unsure if you mean the tool (which is what I am assuming) or if you mean a git repository.